scripts/test-all

```diff
@@ -9,17 +9,17 @@ set -euo pipefail
 # Run from the Makefile environment
 MAKEFILE_RUN=${MAKEFILE_RUN:-""}
 if [[ -z $MAKEFILE_RUN ]] ; then
-  COMMAND="$0 $*" exec make run
+    COMMAND="$0 $*" exec make run
 fi
 
-./scripts/test "$@" -- --nocapture
+./scripts/test "$@"  && echo
```

Comment:
What does the `&& echo` mean?

Reply:
The json output from nextest doesn't have a trailing newline... It's the format from serde_json, so we have to work around it.